src/read.fs

```diff
@@ -8,6 +8,11 @@ open TypeScript.Ts
 open System.Collections.Generic
 open System
 open ts2fable.Naming
+open System.Collections.Generic
+open ts2fable.Syntax
+open System.Collections.Generic
+open Fable
+open System.Collections.Generic
```

Comment:
looks like duplicate

Reply:
This is so stupid
The vscode lonide autocomplete feature regularly automatically add it